ci/check-style.sh

```diff
@@ -51,12 +51,13 @@ replace_original_if_changed() {
 # Apply cmake_format to all the CMake list files.
 #     https://github.com/cheshirekow/cmake_format
 git ls-files -z | grep -zE '((^|/)CMakeLists\.txt|\.cmake)$' |
-  xargs -0 cmake-format -i
+  xargs -P "${NCPU}" -n 1 -0 cmake-format -i
```

Comment:
Did `-n 1` really win?  There are 44 files that currently match that pattern, so, with `NCPU` ~= 8, I've got to imagine that `-n 6` should be better, no?

For what it's worth, I time `-n 1` at about 4.2s and `-n 6` at about 1.4s.

Reply:
I believe it did in my experiments yesterday. But if you find a better combinations, SGTM.

Reply:
I was unable to produce such a big change inside a docker run.  `-n 3` seemed to be the best, but the differentials were fractional seconds, so I'm happy to stick with `-n 1`.